change/@fluentui-react-tabster-24d3044f-d004-4616-89da-62ba9e365ccb.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: Bump tabster to 2.1.0",
```

```suggestion
  "type": "patch",
  "comment": "chore: Bump tabster to 2.1.0",
```

Do we bring new features with this bump?

---

yes the fact that create and dispose are completely changed in React is a feature IMO

---

But are they features that consumers will see?